### ISSUE TITLE
Include alternative views for PRs

### ISSUE THREAD

**Issue by csarven** [solid/specification#410]
Consider providing additional views for GitHub PRs for different audiences. Some possibilities for HTML:

* An HTML preview, e.g., https://htmlpreview.github.io/
* An HTML diff service, e.g., https://services.w3.org/htmldiff

On PR, a bot could update the first comment to include those links (Preview | Diff) at the end of the comment after a separator (`---`). It would be great if the comment can be updated with the new links when there is a change to the PR commit log.

**Comment by kjetilk** [comment]
Yup, and we should add the htmlpreview to the CONTRIBUTION.md, I don't think it is there now?

**Comment by csarven** [comment]
Kjetil, sure thing. CONTRIBUTING currently includes https://github.com/solid/specification/blob/main/CONTRIBUTING.md#self-review :

>Compare your PR's source changes to staging to confirm that the output matches the source and that everything is rendering as expected. The W3C service to [creating diff between HTML pages](https://services.w3.org/htmldiff) can also be used.

and Self-review generally covers previewing.

I think when "Preview | Diff" is autogenerated/updated, it should be sufficient.

Not sure if there is more to say. Did you mean something else about the "htmlpreview to the CONTRIBUTING.md"?

**Comment by kjetilk** [comment]
Mmmmm, I don't quite remember, but I think that since we added a link to htmldiff there, we should add a link to htmlpreview too. But yeah, if we have that bot, that is probably sufficient.
